### PR TITLE
Add section about chaining actions to readme

### PR DESCRIPTION
**Pull request by pactode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,25 @@ class MyController
...
 ```
 
+### Chaining actions
+
+You can chain actions by wrapping them in the `ActionJob`. 
+
+Here's an example of two actions with the same arguments:
+
+```php
+$args = [$userId, $data];
```
Comment (Naoray) [inline README.md]: I'd add `use Spatie\QueueableAction\ActionJob;`

**Comment by brendt** [comment]
Thanks! Can you please fix the merge conflict and take a look at @Naoray suggestion?

**Comment by pactode** [comment]
No problem. I resolved the conflict and added the use statement according to the suggestion.

**Comment by brendt** [comment]
Thanks!

**Pull request merged**